docs/about/release-notes.md

```diff
@@ -15,6 +15,46 @@ You can determine your currently installed version using `mkdocs --version`:
 
 ## Version 0.16 (2016-02-??)
 
+### Major Additions to Version 0.16.0
+
+#### Template variables refactored. (#874)
+
+Page specific variable names in the template context have been refactored as
+defined in [Custom Themes](../../user-guide/custom-themes/#page). The
+old variable names will issue a warning but continue to work for version 0.16,
+but may be removed in a future version.
+
+No global variables were altered except `page_description`. Previously
+its value was altered programicaly based on whether the current 
+page was the homepage. Now it simply maps to `config['site_description']`.
```

Comment:
I think we should do the same thing with the `config`. We pass the config to the templates and a bunch of values from the config. So like you did with the others `copyright` would become `config.copyright` etc. Then we can just deprecate `page_description` and have people use `config.site_description`.